src/config/product.openshiftContainer.js

```diff
@@ -213,7 +214,7 @@ const config = {
               isInline
               component="a"
               variant="link"
-              href={`${helpers.UI_DEPLOY_PATH_LINK_PREFIX}/insights/inventory/${instanceId}/`}
+              href={`${helpers.UI_DEPLOY_PATH_LINK_PREFIX}/insights/inventory/${inventoryId}/`}
```

Comment:
additional updates, maybe?

starting around line `197` and going down to line `209` there is a conditional and ternary that falls back to displaying the `instanceId`... all of it with the intent of being a fallback the user could reference. If `instanceId` is no longer something a user can use to navigate then it should also be updated.

### from
```
{
  [INVENTORY_TYPES.DISPLAY_NAME]: displayName,
  [INVENTORY_TYPES.INVENTORY_ID]: inventoryId,
  [INVENTORY_TYPES.INSTANCE_ID]: instanceId,
  [INVENTORY_TYPES.NUMBER_OF_GUESTS]: numberOfGuests
} = {},
...
if (!instanceId) {
  return displayName;
}

let updatedDisplayName = displayName || instanceId;
```

### to (up to 3 additional updates)
```
{
  [INVENTORY_TYPES.DISPLAY_NAME]: displayName,
  [INVENTORY_TYPES.INVENTORY_ID]: inventoryId,
  [INVENTORY_TYPES.NUMBER_OF_GUESTS]: numberOfGuests
} = {},
...
if (!inventoryId) {
  return displayName;
}

let updatedDisplayName = displayName || inventoryId;
```

This change should also be applied to the other configs that also leverage that fallback.

Reply:
To sum up, stop displaying the instance ID (which changed) and print the inventory ID, so users can use it if needed +100

Reply:
You got it!